android/gradle/libs.versions.toml

```diff
@@ -16,16 +16,16 @@ converter-gson = "2.9.0"
 core-ktx = "1.10.1"
 core-ktx-test= "1.5.0"
 core-testing = "2.2.0"
-cql-engine = "2.4.0"
-cql-evaluator = "2.4.0"
-cql-to-elm = "2.4.0"
+cql-engine = "3.2.0"
+cql-evaluator = "2.6.0"
+cql-evaluator-another = "2.4.0"
 dagger-hilt-compiler = "2.45"
-data-capture = "1.0.0-preview19-SNAPSHOT"
-desugar-jdk-libs = "1.1.5"
+data-capture = "1.0.0-preview20-SNAPSHOT"
+desugar-jdk-libs = "2.0.3"
 easy-rules-jexl = "4.1.0"
 espresso-core = "3.5.1"
 fhir-common-utils = "0.0.10-SNAPSHOT"
-fhir-engine = "0.1.0-beta04-preview4-SNAPSHOT"
+fhir-engine = "0.1.0-beta04-preview5-SNAPSHOT"
 foundation = "1.3.1"
```

Comment:
Would be nice to upgrade all the other non-FHIR dependencies as well as part of this PR

Reply:
Will do so later after resolving the major issues.